### PR TITLE
Carnival references

### PR DESCRIPTION
**Pull request by clee1060**
https://github.com/openstax/cnx-recipes/issues/1152

### CODE REVIEW

File: styles/books/chemistry/book.scss [hunk 1]
```diff
@@ -10,3 +10,5 @@
 @include use('LearningObjective', 'LearningObjectiveSpace');
 @include use('Interconnect', 'carnival:::IconFeatureSpace');
 @include use('Example', 'carnival:::IconFeatureSpace');
+@include use('Appendix', 'carnival:::AppendixSpace');
+@include use('Index', 'carnival:::IndexSpace');
```
Comment (tomjw64) [inline styles/books/chemistry/book.scss]: This seems to be from a different PR, does the base branch need to change?

Comment (clee1060) [reply styles/books/chemistry/book.scss]: Forgot to delete those lines, will do

File: styles/books/anatomy/book.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@import 'framework/framework';
+@import '../../templates/template1/design.scss';
```
Comment (tomjw64) [inline styles/books/anatomy/book.scss]:
```suggestion
@import 'templates/template1/design';
```

File: styles/books/anatomy/book.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+@import '../../templates/template1/settings.scss';
```
Comment (tomjw64) [inline styles/books/anatomy/book.scss]:
```suggestion
@import 'templates/template1/settings';
```

File: styles/designs/carnival/parts/_references-components.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    color: enum('ValueSet:::NULLABLE'),
+    font-family: enum('ValueSet:::NULLABLE'),
+    font-size: enum('ValueSet:::NULLABLE'),
+    font-weight: normal,
+    border-bottom-color: enum('ValueSet:::NOT_NULL'),
+    border-bottom-width: enum('ValueSet:::NOT_NULL'),
+    border-bottom-style: enum('ValueSet:::NOT_NULL'),
+    padding-bottom: enum('ValueSet:::NOT_NULL'),
+  )
+);
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_references-components.scss]:
```suggestion
);

```

File: styles/templates/template1/_settings.scss [hunk 5]
```diff
@@ -6,6 +6,7 @@
 @import 'templates/template1/parts/portrait-settings.scss';
 @import 'templates/template1/parts/appendix-settings.scss';
 @import 'templates/template1/parts/index-settings.scss';
+@import 'templates/template1/parts/references-settings.scss';
```
Comment (tomjw64) [inline styles/templates/template1/_settings.scss]:
```suggestion
@import 'templates/template1/parts/references-settings';
```
If you don't mind making the change for the other imports too, that would be nice :)

File: styles/templates/template1/parts/_references-settings.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            font-family: (_ref: 'typography:::sansFont'),
+            font-size: not-implemented(2.448rem),
+            border-bottom-color: (_ref: 'colorMap:::fontBodyColor'),
+            border-bottom-width: not-implemented(0.1rem),
+            border-bottom-style: solid,
+            margin-bottom: not-implemented(10rem),
+            padding-bottom: not-implemented(1rem),
+        ),
+    ),
+));
```
Comment (tomjw64) [inline styles/templates/template1/parts/_references-settings.scss]:
```suggestion
));

```

File: styles/designs/carnival/parts/_references-components.scss [hunk 7]
```diff
@@ -0,0 +1,27 @@
+// Learning Objectives Components
+$ReferencesTitleComponent: (
+  _name: "TitleContainer",
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_references-components.scss]: Is this really just the container for the title, or for the references as a whole?

File: styles/templates/template1/parts/_references-settings.scss [hunk 8]
```diff
@@ -0,0 +1,26 @@
+@import 'templates/template1/colors';
+@import 'templates/template1/typography';
+@import url('https://fonts.googleapis.com/css?family=IBM+Plex+Sans&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Alegreya&display=swap');
+
```
Comment (helenemccarron) [inline styles/templates/template1/parts/_references-settings.scss]: I think we should import the fonts in the typography file for consistency, at least for now.

File: styles/designs/carnival/parts/_references-components.scss (hunk 7 above)
Comment (clee1060) [reply styles/designs/carnival/parts/_references-components.scss]: Just for the Title

File: styles/templates/template1/_typography.scss [hunk 9]
```diff
@@ -8,12 +8,11 @@
     sansFont: "IBM Plex Sans",
     sansFontLight: "IBM Plex Sans Light",
     sansFontMedium: "IBM Plex Sans Medium",
-    baseFont: "IBM Plex Sans Regular",
+    baseFont: "Alegreya",
```
Comment (helenemccarron) [inline styles/templates/template1/_typography.scss]: This is changing the font for the entire book, is that desired? As a reminder, the base font should be applied on `body`, thus making it the default font for every child of the body (aka, everything)

Comment (clee1060) [reply styles/templates/template1/_typography.scss]: @tomjw64 and I talked to Larissa and she said the baseFont for paragraphs in the whole textbook should be alegreya? So that was why I changed it.

Comment (helenemccarron) [reply styles/templates/template1/_typography.scss]: Ah, got it, didn't realize that was the case (I only read base font in the titles and saw Multi!)

File: styles/books/anatomy/book.scss [hunk 10]
```diff
@@ -0,0 +1,15 @@
+@import 'framework/framework';
```
Comment (helenemccarron) [inline styles/books/anatomy/book.scss]: Please make sure to add the template for this in book.txt (see how chemistry is added)

File: styles/templates/template1/_typography.scss [hunk 11]
```diff
@@ -8,12 +8,11 @@
     sansFont: "IBM Plex Sans",
```
Comment (helenemccarron) [inline styles/templates/template1/_typography.scss]: I see you removed the font imports from the files but I don't see them here, did you include them somewhere else?
